en/docs/guides/login/session-management-logout.md

```diff
@@ -166,14 +166,14 @@ Deploy this sample web app on a web container.
         `                         https://localhost:9443/oauth2/token                       `
     -   **Client Secret:** Enter the client secret of playground2
         application
-        
-    <img name='authorization-code' src='../../assets/img/samples/authorization-code.png' class='img-zoomable' alt='Authorization code'/>
+        iam/docs-is/en/docs/
```

Comment:
what's this?

Reply:
@nilminiwso2  Sorry, mistakenly added!

Fixed and pushed the changes.